### PR TITLE
[Housekeeping] Complete pending TODOs in Core

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Complete pending `TODOs` in Core.

### CODE REVIEW

**Review by rmarinho — CHANGES_REQUESTED** [review]
Build is failing

```
D:\a\_work\1\s\src\Core\src\Platform\iOS\TimePickerExtensions.cs(80,44): error CS1061: 'ITimePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ITimePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Platform\iOS\DatePickerExtensions.cs(137,44): error CS1061: 'IDatePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IDatePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Platform\iOS\TimePickerExtensions.cs(80,44): error CS1061: 'ITimePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ITimePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Platform\iOS\DatePickerExtensions.cs(137,44): error CS1061: 'IDatePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IDatePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBlockExtensions.cs(91,39): error CS1061: 'ILabel' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ILabel' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBoxExtensions.cs(191,39): error CS1061: 'IView' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IView' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(807,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\..\tools\net5.0\..\net472\XamlCompiler.exe" "obj\Release\net6.0-windows10.0.20348\\input.json" "obj\Release\net6.0-windows10.0.20348\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBoxExtensions.cs(191,39): error CS1061: 'IView' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IView' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBlockExtensions.cs(91,39): error CS1061: 'ILabel' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ILabel' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(807,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\..\tools\net5.0\..\net472\XamlCompiler.exe" "obj\Release\net6.0-windows10.0.19041\\input.json" "obj\Release\net6.0-windows10.0.19041\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\src\Core.csproj]
    10 Warning(s)
```

**Comment by jsuarezruiz** [comment]
> Build is failing
> 
> ```
> D:\a\_work\1\s\src\Core\src\Platform\iOS\TimePickerExtensions.cs(80,44): error CS1061: 'ITimePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ITimePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> D:\a\_work\1\s\src\Core\src\Platform\iOS\DatePickerExtensions.cs(137,44): error CS1061: 'IDatePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IDatePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> D:\a\_work\1\s\src\Core\src\Platform\iOS\TimePickerExtensions.cs(80,44): error CS1061: 'ITimePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ITimePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> D:\a\_work\1\s\src\Core\src\Platform\iOS\DatePickerExtensions.cs(137,44): error CS1061: 'IDatePicker' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IDatePicker' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBlockExtensions.cs(91,39): error CS1061: 'ILabel' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ILabel' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBoxExtensions.cs(191,39): error CS1061: 'IView' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IView' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(807,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\..\tools\net5.0\..\net472\XamlCompiler.exe" "obj\Release\net6.0-windows10.0.20348\\input.json" "obj\Release\net6.0-windows10.0.20348\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\src\Core.csproj]
> D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBoxExtensions.cs(191,39): error CS1061: 'IView' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'IView' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> D:\a\_work\1\s\src\Core\src\Platform\Windows\TextBlockExtensions.cs(91,39): error CS1061: 'ILabel' does not contain a definition for 'GetEffectiveFlowDirection' and no accessible extension method 'GetEffectiveFlowDirection' accepting a first argument of type 'ILabel' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj]
> C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(807,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\..\tools\net5.0\..\net472\XamlCompiler.exe" "obj\Release\net6.0-windows10.0.19041\\input.json" "obj\Release\net6.0-windows10.0.19041\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Core\src\Core.csproj]
>     10 Warning(s)
> ```

It seems that we have removed the `GetEffectiveFlowDirection` extension method, any specific reason?

File: src/Core/src/Platform/iOS/TimePickerExtensions.cs [hunk 1]
```diff
@@ -77,7 +77,18 @@ public static void UpdateTime(this MauiTimePicker mauiTimePicker, ITimePicker ti
 
 		public static void UpdateTextAlignment(this MauiTimePicker textField, ITimePicker timePicker)
 		{
-			// TODO: Update TextAlignment based on the EffectiveFlowDirection property.
+			var effectiveFlowDirection = timePicker.GetEffectiveFlowDirection();
```
Comment (hartez) [inline src/Core/src/Platform/iOS/TimePickerExtensions.cs:80]: Are we even sure this needs to be done anymore? Is the alignment of the text actually incorrect when the flow direction is RTL? I strongly suspect that GetEffectiveFlowDirection is no longer needed here.

**Review by hartez — CHANGES_REQUESTED** [review]
Need confirmation that flow direction actually has a problem with these controls.

**Comment by jsuarezruiz** [comment]
To help focus and move faster, we're going to have fewer open PRs and focus on the ones we'll be working on in each release. This PR is not included in the next release so I close it for now. I will reopen it and adapt to changes later.

**Pull request closed without merging**